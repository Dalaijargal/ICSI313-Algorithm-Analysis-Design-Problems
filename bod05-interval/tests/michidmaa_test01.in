100
20 36 10
74 95 5 
55 71 5 
31 83 1 
71 97 5 
75 90 5 
23 74 5 
24 31 10
59 80 1 
68 93 10
46 93 1 
35 44 1 
53 93 5 
26 86 1 
20 97 10
99 99 5 
41 67 1 
46 69 1
38 57 5
43 73 1
52 91 1
1 42 10
61 70 1
23 81 1
4 50 1
20 62 10
27 70 5
2 37 5
20 90 10
42 44 10
2 49 1
36 38 10
5 46 1
9 34 1
18 43 5
46 61 5
52 73 10
57 75 5
83 88 5
32 56 1
6 11 1
37 86 10
39 41 5
3 67 1
7 81 1
36 99 5
11 66 1
59 76 1
19 33 1
77 97 10
18 39 10
64 100 5
40 64 5
58 78 1
24 72 5
56 85 1
71 79 5
42 48 1
1 10 1
72 84 5
40 73 10
18 24 1
9 15 5
90 97 10
50 89 1
38 92 5
57 81 10
14 36 10
38 98 10
13 76 10
1 78 1
8 84 10
58 65 10
72 85 1
33 89 5
58 90 5
89 99 1
76 100 10
28 56 10
35 100 10
25 37 10
64 80 10
26 72 1
36 100 1
48 85 10
29 58 10
27 53 1
32 71 5
36 78 1
17 94 1
39 71 10
1 84 5
4 44 5
0 14 1
27 59 10
57 74 1
0 69 1
7 54 10
6 30 1
19 37 5