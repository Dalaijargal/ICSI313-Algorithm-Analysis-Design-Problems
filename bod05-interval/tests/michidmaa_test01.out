75
0001001000000010100010000000000000001000000000000000000000001000000000000001000000000000010000010000